manifests/C/Chocohead/optifabric/1.x/1.12.x/1.12.10.yaml

```diff
@@ -0,0 +1,52 @@
+# yaml-language-server: $schema=../../../../../../doc/validation-schemas/4.0/version-manifest.json
+
+- loaders:
+    - fabric
+  minecraftVersions:
+    - "1.18"
```

Comment:
Just wondering, did it really not get updated to 1.18.1 lol

Reply:
Or does optifine itself just not have a 1.18.1 release?

Reply:
it does support it, looks like I missed it when copying over the version

Reply:
as much as I want to mess with optifabric users I guess I will fix it =(

Reply:
xD